tests/source/fn_width/100.rs

```diff
@@ -0,0 +1,17 @@
+// rustfmt-config: fn_width_100.toml
+
+impl Trait {
+    fn lorem(first: First, second: Second);
+    fn lorem(first: FirstParameter, second: SecondParameter, third: ThirdParameter);
+    fn lorem(first: FirstParameter, second: SecondParameter, third: ThirdParameter, fourth: FourthParameter);
```

Comment:
Let's expand on these tests cases:

1. use various function declaration headers. e.g: visibility modifiers, unsafe, async, extern etc.
2. functions with generic arguments
3. I'd like to see cases where the parameter types are more complicated than just `First`, Second`, or `ThirdPrameter`. Maybe even make some of them multi-line.
4. Adding test cases where there are comments between the parameters.
5. functions in different contexts:
   - top level functions
   - functions in trait declarations (like you've got here)
   - functions defined in nest modules / blocks

Maybe you can draw some inspiration from some of the bevy code that you're writing?

Reply:
I think most of these are going to be moot since the `fn_param_width` is only going to factor in the length between the parentheses. I can throw them in anyways though.